modules/EmailMan/views/view.config.php

```diff
@@ -164,7 +165,14 @@ public function display()
         if (isset($sugar_config['email_enable_confirm_opt_in']) && $sugar_config['email_enable_confirm_opt_in'] == true) {
             $enableConfirmOptIn = 'CHECKED';
         }
-        $this->ss->assign('EMAIL_ENABLE_CONFIRM_OPT_IN', $enableConfirmOptIn);
```

Comment:
$enableConfirmOptIn variable is unused. value of `$sugar_config['email_enable_confirm_opt_in']` never shows up in settings template anymore?

Reply:
please see updated code.